cmd/crc/cmd/start.go

```diff
@@ -31,6 +31,7 @@ func init() {
 	flagSet.StringP(cmdConfig.PullSecretFile, "p", "", fmt.Sprintf("File path of image pull secret (download from %s)", constants.CrcLandingPageURL))
 	flagSet.IntP(cmdConfig.CPUs, "c", constants.DefaultCPUs, "Number of CPU cores to allocate to the OpenShift cluster")
 	flagSet.IntP(cmdConfig.Memory, "m", constants.DefaultMemory, "MiB of memory to allocate to the OpenShift cluster")
+	flagSet.UintP(cmdConfig.DiskSize, "d", 0, "Total size in GiB of the disk used by the CodeReady Container instance")
```

Comment:
CodeReady Container instance or the OpenShift c;uster? 

Just like memory, disk is NOT assigned to the cluster, but we say so in the help.

Reply:
ah, it's indeed inconsistent, I'll fix.